### DISCUSSION TITLE
Comments with '@' mention username

### DISCUSSION THREAD

**Discussion by Azemblage** [wagtail/wagtail#11155] [Ideas]
Hello,

I have a suggestion that could improve the current 'Comments' feature on the wagtail back office. My idea is to add a functionality that allows users to type '@' and select a user from the system in order to automatically notify them of a specific comment on a page. This would enhance collaboration and ensure that relevant parties are made aware of comments that require their attention.

Furthermore, it would be even more helpful if 'Comments' had a dedicated section on the Dashboard that displays the latest comments that mention a particular user. This would enable the user to quickly access the page and address any outstanding issues.

By implementing these changes, we can make better use of the wagtail back office and facilitate smoother collaboration amongst team members.

**Comment by thibaudcolas** [comment]
Thanks for the suggestion @Azemblage! Makes a lot of sense to me. We have #8325 for the dashboard if you want to add more thoughts about comments there.
